lib/LMSManagers/LMSEventManager.php

```diff
@@ -250,6 +251,10 @@ function GetEventList(array $params) {
         $list2 = array();
         if ($list)
             foreach ($list as $idx => $row) {
+                if(!empty($row['netnodeid'])) {
+                    $row['netnode_name'] = $this->db->GetOne('SELECT name FROM netnodes WHERE id = ?', array($row['netnodeid']));
+                    $row['netnode_location'] = $this->db->GetOne('SELECT address FROM vaddresses WHERE id = (SELECT address_id FROM netnodes WHERE id = ?)', array($row['netnodeid']));
+                }
```

Comment:
Nie bardzo - zamuli bazę danych niepotrzebnymi zapytaniami SQL. Jeśli już proponuję ten sam efekt uzyskać JOIN-ami/LEFT-JOIN-ami w głównym zapytaniu.

Reply:
Zobacz że to się odpala tylko w warunku kiedy jest przypisanie. Myślisz że jak dam to w głównym zapytaniu to zawsze będzie szybciej?

Reply:
Tym bardziej, że masz od razu "na tacy" netnodeid...

Reply:
> Myślisz że jak dam to w głównym zapytaniu to zawsze będzie szybciej?

Nie inaczej.

Reply:
To zrobione :)